src/ol/geom/circle.js

```diff
@@ -85,6 +85,7 @@ ol.geom.Circle.prototype.containsXY = function(x, y) {
 
 
 /**
+ * Return the center of the circle as {@link ol.Coordinate coordinate}.
```

Comment:
This is something we'll probably have to deal with, but the builder client doesn't do any special parsing of JSDoc's custom `@lnk` style annotations.  It would reduce work in the near-term to just have plain text for the first sentence.